web/src/sidebar_ui.js

```diff
@@ -28,6 +28,14 @@ export function hide_streamlist_sidebar() {
     $(".app-main .column-left").removeClass("expanded");
 }
 
+export function any_sidebar_expanded() {
+    if (window.innerWidth > media_breakpoints_num.xl) {
+        // Sidebars are always visible beyond xl breakpoint.
+        return false;
+    }
```

Comment:
Is this first check just an optimization? Might be worth saying that, if so.

Reply:
Added a comment.